Geometry/HGCalCommonData/test/HGCalConvert.cpp

```diff
@@ -149,11 +151,12 @@ class ConvertScintillator {
                  bool debug);
 
   const int layMin_;
+  const int cassette_;
 };
 
 int main(int argc, char* argv[]) {
-  if (argc < 6) {
-    std::cout << "Please give a minimum of 7/5 arguments \n"
+  if (argc < 7) {
```

Comment:
This "7" had to be propagated by hand in several parts of this code: probably better to define a constant for it

Reply:
Given the `return 0` in this if block, I'd say any further checks on `argc < 7` are redundant.